backend/test/images/test-run.sh

```
@@ -262,8 +262,12 @@ while [ "$current_dir" != "/" ]; do
   current_dir="$(dirname "$current_dir")"
 done
 
+# Reports always go to backend/test/end2end/reports regardless of which suite runs
+REPORT_OUTPUT_DIR="${found_test_dir%/$TEST_DIRECTORY}/end2end/reports"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)